### PR TITLE
Add convert_to_bool and convert_to_char

### PR DESCRIPTION
**Pull request by djuggiah**

### CODE REVIEW

**Review by Alex-MobileApps — CHANGES_REQUESTED** [review]
Hi Devesh, the code currently isn't compiling at the moment inside the convert_to_char method at the line `std::cout << c`. I have just noticed a few things that might be some small issues. It looks like the method will also print the char instead of returning the char and the convert_to_bool method may not work if there are uppercase letters. Other than that it is looking good so far. Good stuff!

**Comment by djuggiah** [comment]
Hi Alex, I have updated the code and it's working fine now. Thanks for looking into it.

File: coresdk/src/coresdk/basics.cpp [hunk 1]
```diff
@@ -120,4 +110,25 @@ namespace splashkit_lib
...
+
+    double convert_to_double(const string &text)
+    {
+        return std::stod( text );
+    }
+    
+    char convert_to_char(const string &text)
+    {
+        char c = text.front();
+        return c
```
Comment (Alex-MobileApps) [inline coresdk/src/coresdk/basics.cpp:126]: It looks like this is the line that is stopping the code from compiling

**Review by Alex-MobileApps — CHANGES_REQUESTED** [review]
Hi Devesh, it looks like there is a missing semicolon that is stopping the code from compiling. Please see the attached comment with where the line is located.

File: coresdk/src/coresdk/basics.cpp [hunk 2]
```diff
@@ -120,4 +110,25 @@ namespace splashkit_lib
...
+    char convert_to_char(const string &text)
+    {
+        char c = text.front();
+        return c;
+    }
+
+    bool convert_to_bool(const string &text)
+    {   
+        string s = to_lowercase(trim(text));
+        return s == "true" || s == "t" || s == "1";
```
Comment (timmydzeko) [inline coresdk/src/coresdk/basics.cpp:132]: Will this method convert anything that isn't those three terms to False? Do we want a 5 that is put into the convert to bool to return false?

A suggested way to do this is to put the code within an if statement, e.g.
```
{
    if (is_bool(text) {
         // convert to bool body added here
    }
}
```

Comment (timmydzeko) [reply coresdk/src/coresdk/basics.cpp:132]: @Alex-MobileApps Is this necessary? Is there a downside to turning everything else into a false value?

Comment (djuggiah) [reply coresdk/src/coresdk/basics.cpp:132]: Yes anything else other than these 3, it will out false. Do I still have to include your suggested code?

**Pull request closed without merging**